### PR TITLE
Token address processing fixes

### PR DESCRIPTION
**Pull request by MrRoudyk**
## Fixed calculation of token utilization statistics in `get_utilization_stats` (apps/dashboard_app/helpers/protocol_stats.py):
- Added handling of division by zero cases to prevent exceptions and ensure correct utilization metrics calculation
- Simplified and optimized function code

## Added token address normalization using `add_leading_zeros` (apps/dashboard_app/helpers/tools.py):
- Added consistent address format handling
- Prevented errors related to address format mismatches during data processing

## Status of known issues:
- ✅ Fixed issue with STRK token (`0x4718f5a0fc34cc1af16a1cdee98ffb20c31f5cd61d6ab07201858f4287c938d`) by normalizing address format
- ⚠️ Error for DAI token (`0x00da114221cb83fa859dbdb4c44beeaa0bb37c7537ad5ae66fe5e0efd20e6eb3`) persists as this address is missing from API response

### CODE REVIEW

File: apps/dashboard_app/helpers/protocol_stats.py [hunk 1]
```diff
@@ -248,35 +257,48 @@ def get_utilization_stats(
...
-    data[utilization_columns] = data[utilization_columns].apply(lambda x: x.round(4))
-    # data[utilization_columns] = data[utilization_columns].map(lambda x: round(x, 4))
+
+    total_debt = general_stats["total debt (usd)"].astype(float)
+    total_supply = supply_stats["total supply (usd)"].astype(float)
+    total_utilization = total_debt / (total_debt + total_supply)
+    total_utilization = total_utilization.replace([np.inf, -np.inf], 0).fillna(0)
+    data["Total utilization"] = total_utilization.round(4)
+
+    tokens = ["eth", "wbtc", "usdc", "dai", "usdt", "wsteth", "lords", "strk"]
```
Comment (djeck1432) [inline apps/dashboard_app/helpers/protocol_stats.py:289]: Shouldn't be by default uppercase, `STRK utilization` and `"STRK supply"` and `"STRK debt"` everywhere is uppercase

File: apps/dashboard_app/helpers/protocol_stats.py [hunk 2]
```diff
@@ -116,7 +121,11 @@ def get_supply_stats(
     df = pd.DataFrame(data)
     df["Total supply (USD)"] = sum(
         df[column]
-        * Decimal(prices[TOKEN_SETTINGS[column.replace(" supply", "")].address])
+        * Decimal(
+            prices[
+                add_leading_zeros(TOKEN_SETTINGS[column.replace(" supply", "")].address)
```
Comment (djeck1432) [inline apps/dashboard_app/helpers/protocol_stats.py:126]: Didn't it effect other tokens? I remember that there was issue with 2 of tokens?

**Review by djeck1432 — CHANGES_REQUESTED** [review]

File: apps/dashboard_app/helpers/protocol_stats.py (hunk 1 above)
Comment (MrRoudyk) [reply apps/dashboard_app/helpers/protocol_stats.py:289]: The problem was with WBTC: in `supply_stats`, the column is `WBTC`, but in `debt_stats`, it is `wBTC`, or the other way around. To fix this, I changed all column names to lowercase

File: apps/dashboard_app/helpers/protocol_stats.py (hunk 2 above)
Comment (MrRoudyk) [reply apps/dashboard_app/helpers/protocol_stats.py:126]: I don’t see any errors on my database. The issue might be related to a specific dataset

**Review by djeck1432 — APPROVED** [review]
Looks good to me

**Pull request merged**